File: jmolecules-axon/02-axonstyle-cqrs-es-bank/src/main/java/org/jmolecules/example/axonframework/AxonBankApplication.java
package org.jmolecules.example.axonframework;

import io.swagger.v3.oas.annotations.OpenAPIDefinition;
import org.springframework.boot.SpringApplication;
import org.springframework.boot.autoconfigure.SpringBootApplication;

@SpringBootApplication
@OpenAPIDefinition
public class AxonBankApplication {

  public static void main(String[] args) {
    SpringApplication.run(AxonBankApplication.class, args);
  }

}

File: jmolecules-axon/03-onion-axon-cqrs-es-bank/src/main/kotlin/org/jmolecules/example/axonframework/AxonBankApplication.kt
package org.jmolecules.example.axonframework

import io.swagger.v3.oas.annotations.OpenAPIDefinition
import org.springframework.boot.SpringApplication
import org.springframework.boot.autoconfigure.SpringBootApplication
import org.springframework.boot.runApplication


@SpringBootApplication
@OpenAPIDefinition
class AxonBankApplication

fun main(args: Array<String>) {
    runApplication<AxonBankApplication>(*args).let { Unit }
}
